incubator/java-openliberty/image/project/validate.sh

```diff
@@ -29,14 +20,20 @@ p_groupId=$(xmlstarlet sel -T -N x="http://maven.apache.org/POM/4.0.0" -t -v "/x
 p_artifactId=$(xmlstarlet sel -T -N x="http://maven.apache.org/POM/4.0.0" -t -v "/x:project/x:parent/x:artifactId" pom.xml)
 p_version_range=$(xmlstarlet sel -T -N x="http://maven.apache.org/POM/4.0.0" -t -v "/x:project/x:parent/x:version" pom.xml)
 
+# Install parent pom
+echo "Installing parent ${a_groupId}:${a_artifactId}:${a_version}"
+mvn install -Dmaven.repo.local=/mvn/repository -Denforcer.skip=true -f ../pom.xml
+
+
+
 # Check child pom for required parent project
 if [ "${p_groupId}" != "${a_groupId}" ] || [ "${p_artifactId}" != "${a_artifactId}" ]; then
   echo "Project pom.xml is missing the required parent:
 
   <parent>
     <groupId>${a_groupId}</groupId>
     <artifactId>${a_artifactId}</artifactId>
-    <version>${a_range}</version>
```

Comment:
This is just something we hadn't realized was wrong?  (I don't see it set anywhere)?   I guess it's just an error msg though.  Good catch.

Reply:
Right, probably a typo. The parent doesnt have a range for it's version, so we can only recommend a set version number.